### PR TITLE
6943 add new users to message modal

### PR DESCRIPTION
**Pull request by mrinsin**
image

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/presenter/computeds/workQueueSectionHelper.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ export const workQueueSectionHelper = (get, applicationContext) => {
         chambers: 'Chambers',
         clerkofcourt: 'Clerk of the Court',
         docket: 'Docket',
+        floater: 'Floater',
         petitions: 'Petitions',
+        reportersOffice: "Reporter's Office",
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/workQueueSectionHelper.js:14]: We'll want to replace the single-quote with an apostrophe here.

**Review by kkoskelin — APPROVED** [review]
Once apostrophe is added, 👍

File: web-client/src/presenter/computeds/workQueueSectionHelper.js (hunk 1 above)
Comment (mrinsin) [reply web-client/src/presenter/computeds/workQueueSectionHelper.js:14]: Thanks!

Comment (mrinsin) [reply web-client/src/presenter/computeds/workQueueSectionHelper.js:14]: Mind if I do it in the next PR for this story?

**Pull request merged**